app/src/main/java/com/hippo/ehviewer/ui/scene/ProgressScene.java

```diff
@@ -96,13 +99,12 @@ private boolean doJobs() {
             if (mGid == -1 || mPToken == null || mPage == -1) {
                 return false;
             }
-
-            EhRequest request = new EhRequest()
-                    .setMethod(EhClient.METHOD_GET_GALLERY_TOKEN)
-                    .setArgs(mGid, mPToken, mPage)
-                    .setCallback(new GetGalleryTokenListener(context,
-                            activity.getStageId(), getTag()));
-            EhApplication.getEhClient(context).execute(request);
```

Comment:
This code requests the gallery token for a page token, to open a gallery detail activity from the gallery page URL, like https://exhentai.org/s/783d5c21dd/1140853-1

Reply:
Should it be an exception only when Request to first page?